src/test/java/net/jqwik/micronaut/ApplicationPropertyTest.java

```diff
@@ -39,17 +39,21 @@ public Map<String, String> getProperties() {
 
     @Property(tries = 1)
     @io.micronaut.context.annotation.Property(name = "test.method.property", value = "Hello method!")
+    @io.micronaut.context.annotation.Property(name = "test.method.property2", value = "Hello method2!")
```

Comment:
Can you explain why this method is injecting another property? I can't understading the idea behind it and I want to make sure I don't miss any important detail.

Reply:
I added the second property to show that repeatable annotations work, because my initial approach did not.